insights/parsers/abrt_status_bare.py

```diff
@@ -0,0 +1,25 @@
+"""
+AbrtStatusBare - command ``/usr/bin/abrt status --bare=True``
+=============================================================
+
+``/usr/bin/abrt status --bare=True`` returns the number of problems ABRT
+detected in the system.
+
+Examples:
+    >>> status.problem_count
+    0
```

Comment:
@ernestask please include testing for your docstring examples as documented [here](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#docstring-details).